### DISCUSSION TITLE
📚以後作筆記就靠HackMD啦!數學公式想寫的漂漂亮亮、想做PPT他全都罩

### DISCUSSION THREAD

**Discussion by JunTingLin** [1101-GS4719-C/Python-Community-GS4719-C#516] [General]
# 📚以後作筆記就靠HackMD啦!數學公式想寫的漂漂亮亮、想做PPT他全都罩

## 你知道HackMD是從大學的資案作業誕生的嗎❓
HackMD 主打即時文件協作（real-time collaboration），2014 年底由當時還在台北科技大學資工系唸碩士的吳承翰開發，最初是學校資安課的期末作業。一直到 2017 年 3 月正式登記為公司前，都只是吳承翰個人獨立開發與維護的 side-project。現在團隊有 6 人，執行長楊宗鎧是吳承翰大學實習時所認識的前輩，具財務背景；他自己則擔任技術長。😎

## HackMD是啥
HackMD 是讓你快速撰寫筆記(使用MarkDown語法)，流暢管理知識，支援多種衍生功能的生產力工具，最重要的事還可以一起多人線上共筆💻

## 數學符號等想用電腦打，但word太難用，靠他
下面這張圖就是我同學的統計作業，用打的就很整齊，而且還可以互相討論💦
![螢幕擷取畫面 2022-03-22 002936](https://user-images.githubusercontent.com/92431095/159692295-ba59e71e-111e-4ef7-8ee2-7b31d1874633.png)

詳細LaTeX 數學語法說明可以參考 [這位大神的筆記](https://hackmd.io/@sysprog/B1RwlM85Z)

## 用HackMD做簡報免煩惱
HackMD 整合了 reveal.js，讓我們可以用 Markdown 輕鬆寫簡報，下面這張圖來自學校程式設計研究社，左邊是講者寫的稿，右邊就是簡報預覽畫面啦(可以切上下左右)
![螢幕擷取畫面 2022-03-22 001010](https://user-images.githubusercontent.com/92431095/159692244-d35c4b2e-ff25-4042-a9f9-1b1b21cf2c8a.png)
![螢幕擷取畫面 2022-03-22 001054](https://user-images.githubusercontent.com/92431095/159692259-3621fd97-2f7d-4372-93d5-a1d0fe71a7bb.png)

詳細的製作簡報可以參考 [這位大神的筆記](https://hackmd.io/s/how-to-create-slide-deck-tw)

## 不只這樣，我還可以跟github同步
這樣我的筆記也有版本紀錄啦，但很可惜的事目前一段時間內只能push上去指定的次數(我記得好像2️⃣0️⃣次吧~~)

### 如何同步呢
其實非常簡單，按右上角三個點，選擇版本同步
![螢幕擷取畫面 2022-03-21 235521](https://user-images.githubusercontent.com/92431095/159691956-90a8c4bd-7f81-4afb-8c33-ecace20eab40.png)

授權github帳號
![螢幕擷取畫面 2022-03-21 235548](https://user-images.githubusercontent.com/92431095/159691997-599da082-1e3b-4f23-a051-767dea6d11a9.png)
![螢幕擷取畫面 2022-03-21 235619](https://user-images.githubusercontent.com/92431095/159692001-64a51c93-1d81-47d0-9003-516e3099b41e.png)

推送至github
![螢幕擷取畫面 2022-03-21 235910](https://user-images.githubusercontent.com/92431095/159692097-9ce699d0-419c-4376-a6a7-ef6467f4e40c.png)

選擇要推送至哪個儲存庫
![螢幕擷取畫面 2022-03-22 004026](https://user-images.githubusercontent.com/92431095/159692128-d014de8b-8da8-4290-88da-bbc2e4513bd6.png)

這樣就大功告成啦

# 參考資料
+ [為工程師文件而生的協作平台：HackMD 開發故事](https://medium.com/starrocket/hackmd-product-story-1e332f83d343)

**Comment by hapiwangy** [comment]
HackMD的出身僅僅是一個期末作業，但當越來越多人使用、維護、開發之後，他也發展成了現在一個大家都很常使用的工具。期望將來他能夠造福更多的人。

**Comment by lx3211** [comment]
建立新環境的時候指令步驟也可以用Hackmd記錄下來
這樣之後換新電腦的時候就不用重新查了

**Comment by xkllkx** [comment]
謝謝分享，之後有時間可以嘗試看看。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，原來有這麼棒的工具。

**Comment by Yuhsin000** [comment]
對打數學公式感到很困擾，以後作筆記用HachMD試試

**Comment by ghost** [comment]
先ca這東西，這對我來說感覺很需要

**Comment by PoooYen** [comment]
對理工學院的同學真的是一大福音，不然常常要打特殊符號的時候找到眼睛痛

**Comment by awkward-willy** [comment]
HackMD 真的是相當方便的工具
不論是用簡報或是共享的功能都很好用
加上支援 markdown 語法
能更快速地完成想要的表示內容
甚至不用付費就能享有大部分的功能 大推

**Comment by sys511613** [comment]
這個東西真的很好用
在裡面加上tag就可以管理多篇筆記
裡面還有一些有趣的小功能
蠻好玩的
![image](https://user-images.githubusercontent.com/73024991/171339826-63d00370-4ce1-4a40-994d-2e67be31c46f.png)

**Comment by Rosancl** [comment]
好用推推，比自己手寫整齊好看很多~~
